adhocracy4/polls/static/PollDetail/PollQuestion.jsx

```diff
@@ -48,11 +48,12 @@ export const PollQuestion = (props) => {
   const handleMultiChange = (event, isOther) => {
     const choiceId = parseInt(event.target.value)
     setUserChoices(prevState => [...prevState, choiceId])
+    setOtherChoiceAnswer(otherChoiceAnswer)
```

Comment:
@khamui when editing your choice and updating multiselect checkbox when the other option is set the validation fails because the other text has not been updated and so is 'empty' according to validation but if we set it when the multiselect is updated then it is not empty according the validation and so works